src/image-resizer/image-resizer.js

```diff
@@ -211,7 +211,7 @@ export class ImageResizerCustomElement {
           -x, -y, resized[0], resized[1],
           0, 0, this.canvas.width, this.canvas.height);
 
-        this.output = this.canvas.toDataURL(this.type, this.encoderOptions);
+        this.output = this.canvas.toDataURL(this.type, Number(this.encoderOptions));
```

Comment:
If no `encoderOptions` attribute is passed, value will be 0 : `Number(undefined) === 0`. Please test if value is a string before transforming it to number.